doc/conf.py

```diff
@@ -61,26 +61,25 @@
     "requires": requirements,
 }
 
-# MyST-NB configurations.
+# MyST-NB configurations
 # Reference: https://myst-nb.readthedocs.io/en/latest/configuration.html
-nb_render_markdown_format = "myst"  # The format to use for text/markdown rendering
+nb_render_markdown_format = "myst"  # Format for text/markdown rendering
 
-# Make the list of returns arguments and attributes render the same as the
-# parameters list
+# Render the return argument and attribute lists in the same way as the parameter lists
 napoleon_use_rtype = False
 napoleon_use_ivar = True
 
-# sphinx_auto_typehints
+# Configure sphinx_auto_typehints
```

Comment:
Maybe make all these consistent?
```
# MyST-NB configurations
# Configure intersphinx
# Configure options for sphinx-copybutton
# Theme configuration
```

Reply:
Done in commit https://github.com/GenericMappingTools/pygmt/pull/3573/commits/6737a7d0a5484c1d0e251911fd8fa76d96087620.